### ISSUE TITLE
dcdplugin) Warning: DCD header claims 16487 frames, file size indicates there are actually 16486 frames

### ISSUE THREAD

**Issue by ZixuanFeng-NYU** [openmm/openmm#4409]
My job was terminated abnormally due to exceeded disk quota, so I think the last frame was not written fully, but how can I only read the fully/successfully written frames? I didn't see any hyperparameter allowing me to read specific number of frames

**Comment by peastman** [comment]
I think you're asking in the wrong place.  OpenMM doesn't have any code for reading DCD files, only writing them.  Whatever code you're using to read the file, you need to ask the authors of it.

**Comment by ZixuanFeng-NYU** [comment]
Yes but I think openMM should not write how many frames at the beginning, for example what if someone wants to read an unfinished simulation for just to see what happened in the frames already been saved.

**Comment by peastman** [comment]
The number of frames *must* be in the header.  That's part of the format.  It updates the header every time it writes a new frame.
